processor-samples/streamlistener-basic/README.adoc

```diff
@@ -66,6 +66,6 @@ All the instructions above apply here also, but instead of running the default `
 
 * `./mvnw clean package -P rabbit-binder`
 
-* `java -jar target/streamlistener-basic-0.0.1-SNAPSHOT.jar`
+* `java -jar target/streamlistener-basic-0.0.1-SNAPSHOT-rabbit.jar`
 
-Once you are done testing: `docker-compose -f docker-compose-rabbit.yml down`
\ No newline at end of file
+Once you are done testing: `docker-compose -f docker-compose-rabbit.yml down`
```

Comment:
What change did you do in this line, please?

Reply:
Nothing. I edit file using github. I guess It's some styling change. It's not important for this PR

Reply:
I guess there is a trailing whitespace in the end of this line after editing here on GitHub.
Would be great to have it removed.

Although this might just not have value to pursue.

Let's merge it!

/CC @sobychacko

Reply:
LGTM - merging.